### ISSUE TITLE
Get the probabilities of each class

### ISSUE THREAD

**Issue by alexdess** [NielsRogge/Transformers-Tutorials#65]
once the outputs are obtained
`outputs = model(**encoded_inputs)`

What is the easiest way to get the probabilities of each class? I would indeed like to define a prediction confidence using probability.

**Comment by NielsRogge** [comment]
Hi,

So deep learning models typically return "logits", also called "unnormalized scores". These are just the raw scores that the model gives for each class, for each example in a batch. It's a tensor of shape (batch_size, num_labels).

If you're doing binary/multi class classification, then you can apply a softmax row-wise to turn them into probabilities, like so:

```
import torch

logits = outputs.logits
probabilities = torch.nn.functional.softmax(logits, dim=-1)
```
This is still a tensor of shape (batch_size, num_labels), but now it contains normalized scores (in other words, probabilities) for every example in the batch. So to get the probabilities for the first example in batch, you can do probabilities[0,:].
